sdp-utils.c

```diff
@@ -1033,6 +1035,11 @@ janus_sdp *janus_sdp_generate_offer(const char *name, const char *address, ...)
 			janus_sdp_attribute *a = janus_sdp_attribute_create("rtpmap", "%d %s", dtmf_pt, janus_sdp_get_codec_rtpmap("dtmf"));
 			m->attributes = g_list_append(m->attributes, a);
 		}
+		/* Check if there's a custom fmtp line to add for audio */
+		if(audio_fmtp) {
```

Comment:
This read may be of an uninitialized `audio_fmtp`, it should set it to NULL as is done for `audio_codec` and `video_codec`.

Reply:
Good catch, thanks! Fixing now :+1: